### PR TITLE
fix: set default aria-label for breadcrumb nav

### LINKED ISSUE #2185: Set default aria-label for breadcrumb
## Detailed description

In `ibm-breadcrumb` component, breadcrumb `nav` tag and overflow menu should have default `aria-label` value same as mention in [carbon design](https://carbondesignsystem.com/components/breadcrumb/accessibility/)

This is  a bug?

Related to `<ibm-breadcrumb>`

Codesandbox - [ibm-breadcumb](https://angular.carbondesignsystem.com/?path=/story/components-breadcrumb--model)

## Additional information

FYI :- Carbon React uses 'Breadcrumb' by default [ref](https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/Breadcrumb/Breadcrumb.js#L32)

### PR DESCRIPTION
**Pull request by kush-savani**
Closes #2185 

Set default aria-label for breadcrumb nav

#### Changelog

**New**

* add new `BREADCRUMB` key in `en.ts` file.
* Set default `aria-lable` for breadcrumb component.

**Before**
Screenshot 2022-07-14 at 9 34 21 AM

**After**
Screenshot 2022-07-14 at 9 33 42 AM

### CODE REVIEW

**Comment by Akshat55** [comment]
Will wait for credits to be added to travis before merging.

**Review by Akshat55 — APPROVED** [review]

File: src/breadcrumb/breadcrumb.component.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { BreadcrumbItem } from "./breadcrumb-item.interface";
 import { BreadcrumbItemComponent } from "./breadcrumb-item.component";
 import { DomSanitizer } from "@angular/platform-browser";
```
Comment (Akshat55) [inline src/breadcrumb/breadcrumb.component.ts:15]:
```suggestion
```

Could you remove DomSanitizer import & dependency injection in constructor as well, please. 

It isn't being used anywhere in the code & seems to have remained from previous changes. Thanks!

File: src/breadcrumb/breadcrumb.component.ts [hunk 2]
```diff
@@ -181,7 +182,11 @@ export class Breadcrumb implements AfterContentInit {
 	protected _threshold: number;
 	protected _skeleton = false;
 
-	constructor(protected domSanitizer: DomSanitizer, @Optional() protected router: Router) { }
+	constructor(
+		protected domSanitizer: DomSanitizer,
+		protected i18n: I18n,
+		@Optional() protected router: Router
+	) { }
```
Comment (Akshat55) [inline src/breadcrumb/breadcrumb.component.ts:189]:
```suggestion
	constructor(protected i18n: I18n, @Optional() protected router: Router) { }
```

**Review by Akshat55 — CHANGES_REQUESTED** [review]

**Review by Akshat55 — APPROVED** [review]
Code is good to go, just need to fix DCO

**Review by Akshat55 — APPROVED** [review]

**Review by zvonimirfras — APPROVED** [review]

**Pull request merged**